schemes/chemistry/prescribed_aerosol_deposition_flux.F90

```diff
@@ -0,0 +1,326 @@
+! Manages reading and interpolation of prescribed aerosol deposition
+! fluxes. These are the deposition fluxes sent to the surface model.
+!
+! Note: this module currently only implements bulk aerosol deposition fluxes.
+!
+! This scheme requires the concurrent use of prescribed_aerosols scheme
+! to provide prescribed aerosol concentrations.
+!
+! Based on original CAM version from: Francis Vitt
+module prescribed_aerosol_deposition_flux
+  use ccpp_kinds, only: kind_phys
```

Comment:
I might move this `use` statement down to the subroutine scope, as it doesn't appear that any module-level variable needs it.